canonical-websites/cn.ubuntu.com
# Convert /internet-of-things to v1 styles

**Note**: This branch is built on the changes in https://github.com/canonical-websites/cn.ubuntu.com/pull/152 - that PR should be landed before this one.

## Done

- Converted /news section of the site to Vanilla v1 styles

## QA

- Pull code
- Sanity check file deletions
- Run `./run serve --watch`
- View desktop page on http://0.0.0.0:8010/internet-of-things
- Check page layout looks good

Fixes https://github.com/ubuntudesign/vanilla-squad/issues/69
